test/local/rns.available.test.ts

```diff
@@ -21,14 +22,18 @@ import { Options } from '../../src/types';
 import { labelhash } from '../../src/utils';
 import { ZERO_ADDRESS } from '../../src/constants';
 
+const web3Instance = web3 as unknown as Web3;
+const rsk3 = new Rsk3(web3.currentProvider);
 
-describe('subdomains.available', () => {
+describe.each([
+  web3Instance,
+  rsk3,
+])('subdomains.available', (provider) => {
```

Comment:
I think the term provider here may be overloaded, if the variables are named `web3Instance` and `rsk3Instance`, we should use `instance` here instead.